docs/book/releases/v2.1.0.md

```diff
@@ -0,0 +1,66 @@
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - v2.1.0 release
+
+## New Feature
+
+- CSI Migration for in-tree vSphere volumes on vSphere 7.0 Update 1. This feature requires upgrading Kubernetes to 1.19 release.
+
+## Deployment files
+
+- https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/release-2.1/manifests/v2.1.0
+
+## Minimum Kubernetes Version
+
+- v1.17.0
```

Comment:
The statement in the beginning of this page says that the CSI migration feature requires upgrading k8s to 1.19 release. This is confusing to the reader.

Reply:
added one more buttet to clarify this.

- For vSphere CSI Migration feature minimum Kubernetes Version requirement is v1.19.0


v1.17.0 comes from the side car container requirement.